resources/css/bem/user-search-card.less

```diff
@@ -71,10 +72,16 @@
 
     &--username {
       .link-plain();
+
+      display: inline-block;
+      white-space: preserve nowrap;
```

Comment:
apparently just `pre` is enough? (that's also what `.u-ellipsis-pre-overflow` does)

Reply:
weird, it didn't truncate before, then again I did more stuff after it 🤔